### DISCUSSION TITLE
Function

### DISCUSSION THREAD

**Discussion by Walterhart** [supabase/supabase#10570] [Questions] [answered]
I am trying to make a functiont that pass a video_id as an argument and return all videos based on that argument.  The issue is I am new at function and my query returning all videos. Is it because of my return type? 
![image](https://user-images.githubusercontent.com/70658679/204220298-ab5d72a5-52b6-4e05-a492-57d3016bde04.png)

![image](https://user-images.githubusercontent.com/70658679/204219779-42ab9b0e-ff7f-4b18-995a-dfbf51fb7bbd.png)

![image](https://user-images.githubusercontent.com/70658679/204220196-7c51ab20-0c46-4b0f-961d-9266f3d57011.png)

**Comment by GaryAustin1** [comment, marked as answer]
Try renaming your parameter.  Might be confused as it is same name as column.

**Reply by Walterhart** [reply]
Thanks that was the issue!
